index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Alice Cieslik . Persönliche Website</title>
+  </head>
+  <body>
+    <header>
+      <p><strong>Alice Cieslik . Persönliche Website</strong> </p>
+
+      <p>Ich baue eine <strong>Website</strong>.</p>
+    </header>
+
+    <nav>
+      <strong>Navigation</strong> 
+      <ul>
+        <li>Home</li>
+        <li>Über mich</li>
+        <li>Dienstleistungen</li>
+        <li>Blog</li>
+        <li>Kontakt</li>
+      </ul>
+    </nav>
+
+    <main>
+      <h1>Hier steht der Hauptinhalt</h1>
+      <h2>Home</h2>
+
+      <p> Lorem ipsum, <br> dolor sit amet consectetur adipisicing elit. Aliquam similique nam expedita fugit porro doloremque minus? Consequuntur magni laborum magnam accusantium voluptatem! Vitae non possimus iure cupiditate quo maxime inventore?
+      </p>
+
+      <h2>Über mich</h2>
+
+      <p> Lorem ipsum, <br> dolor sit amet consectetur adipisicing elit. Aliquam similique nam expedita fugit porro doloremque minus? Consequuntur magni laborum magnam accusantium voluptatem! Vitae non possimus iure cupiditate quo maxime inventore?
+    </p>
+
+    <h2>Dienstleistungen</h2>
+    <p>
+      <ol>
+        <li>Eins</li>
+        <li>Zwei</li>
+        <li>Drei</li>
+        <li>sortierte Liste</li>
+      </ol></p>
+
+    <h2>Blog</h2>
+
+    <p>Kurzvorschau aktuelle Artikel</p>
+
+    <h2>Kontakt</h2>
+
+  
```

Comment:
I recommend you to structure you content with `section` to group all elements that are part of one topic of your personal website. [See for further information the article about the section on MDN.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)